tests/docker_service_fixtures.py

```diff
@@ -53,7 +54,7 @@ def __init__(self, worker_id: str) -> None:
         self._base_command = [
             "docker",
             "compose",
-            "--file=tests/docker-compose.yml",
+            f"--file={Path(__file__).parent / 'docker-compose.yml'}",
```

Comment:
this is unrelated but I found it annoying to have in Pycharm to change my working directory to make this work, now you can run tests using that compose file from wherever working dir you want